backend/src/appointment/controller/mailer.py

```diff
@@ -168,10 +168,10 @@ def text(self):
 
 
 class ConfirmationMail(Mailer):
-    def __init__(self, confirm_url, deny_url, attendee, date, *args, **kwargs):
+    def __init__(self, confirm_url, deny_url, attendee_name, attendee_email, date, *args, **kwargs):
         """init Mailer with confirmation specific defaults"""
-        self.attendee = attendee
-        self.attendee.name = self.attendee.name.title()
+        self.attendee_name = attendee_name.title()
```

Comment:
I suppose we don't need the title() call anymore:

```suggestion
        self.attendee_name = attendee_name
```